src/Gamepage/Gamepage.js

```diff
@@ -68,18 +69,18 @@ export class Gamepage extends Component {
     return (
       <div className="game-page" data-testid="game-page">
         <div className="hints">
-          <h2>Hints</h2>
+          <h2><em>Hints</em></h2>
```

Comment:
What do all these `<em>` tags do?

Reply:
They're for emphasis, from what I know. I was wondering if we needed this inside the <h1> tag...